### DISCUSSION TITLE
Typescript SDK for Webhooks

### DISCUSSION THREAD

**Discussion by hhelmric** [codatio/codat-docs#526] [Q&A]
For the Typescript SDK, I see there is a class for `PushOperationStatusChangedWebhook` (among other related Webhook classes) which show keys as camelCase:
![image](https://github.com/codatio/codat-docs/assets/25378854/14a3e985-a6e4-4f91-8d98-d417d130cfa1)

However, the data I am getting from the web hook has slightly different keys (the first letters are capitalized)
![image](https://github.com/codatio/codat-docs/assets/25378854/c2e22089-55bb-40a5-9733-9d359c5213b5)

https://github.com/codatio/client-sdk-typescript/blob/05f065f18a30ff9562ceaed4ff7c523f9034d3d4/common/src/sdk/models/shared/pushoperationstatuschangedwebhook.ts

**Comment by dcoplowe** [comment]
Hi Harry, 

Thank you for raising this. Let me take a look at our implementation and I'll get back to you once we have a fix. 

Many thanks,

David

**Comment by dcoplowe** [comment]
Hi Harry, 

A fix for this has been released. Once again, thank you for raising the issue and do let us know if you experience any further problems. 

Many thanks, 

David
